src/pages/ThorChainLP/components/AddLiquitity/AddLiquidityInput.tsx

```diff
@@ -375,7 +383,7 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
     ...reactQueries.mutations.approve({
       assetId: asset?.assetId,
       spender: inboundAddressData?.router,
-      from: allowanceFromAddress,
+      from: poolAssetAccountAddress,
```

Comment:
Unified into `poolAssetAccountAddress` - since we do need  a from address for all pool assets, not only for EVM tokens in the context of approvals, we don't need `allowanceFromAddress` anymore as that's effectively duplicate logic

Reply:
🧠